metro/proto/v1/spec.proto

```diff
@@ -1038,4 +1050,10 @@ message ListProjectSubscriptionsRequest {
 message ListProjectSubscriptionsResponse {
     // The names of subscriptions created in the specified project.
     repeated string subscriptions = 1;
-}
\ No newline at end of file
+}
+
+message GetSubscriptionRequest {
+    // Required. The subscription from which messages should be pulled.
+    // Format is `projects/{project}/subscriptions/{sub}`.
+    string subscription = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
should we rename this to name may be, since from the message context itself, it is clear that its subscription name